src/tail/mod.rs

```diff
@@ -33,3 +62,16 @@ impl Tail {
         })
     }
 }
+
+/// handle_sigint simply waits on a ctrl_c from the system and sends messages to each registered
+/// transmitter when it is received.
+async fn listen_for_sigint(txs: Vec<oneshot::Sender<()>>) -> Result<(), failure::Error> {
+    tokio::signal::ctrl_c().await?;
+    for tx in txs {
+        // if send errors, it is because the receiver has been de-allocated, and that's okay
```

Comment:
this comment doesn't _quite_ make sense to me. i think i get it but might want some clearer wording

Reply:
if the receiver goes out of scope, it's likely due to the task returning early for some reason, in which case we don't need to tell it to shut down because it already has. will use this verbiage in the comment.